proto/v1/branch_service.proto

```diff
@@ -55,6 +55,15 @@ service BranchService {
     option (google.api.method_signature) = "name";
   }
 
+  // DiffBranchDatabases works similar to branch rebase.
+  // 1) set the base as the schema of a database;
+  // 2) apply the changes between base and head of branch to the new base (schema of database);
+  // 3) resolve these conflicts manually if any.
+  rpc DiffBranchDatabases(DiffBranchDatabasesRequest) returns (DiffBranchDatabasesResponse) {
+    option (google.api.http) = {post: "/v1/{name=instances/*/databases/*}:mergeDatabases"};
```

Comment:
Maybe we should put it in the `database_service.proto`? (and the API should be `/instances/*/databases/*:mergeBranch`

Reply:
And if it's only diff (not really merge) maybe should be `/instances/*/databases/*:diffBranch`

Reply:
Updated the API to use branch as name. Also added comments about future steps for conflict schema.